### PR TITLE
Don't sleep()

### PR DESCRIPTION
**Pull request by bhs**
Regardless of whether the sleeping can ever happen on the main thread, non-blocking dispatch_after() is a better practice.

### CODE REVIEW

**Comment by bhs** [comment]
cc @eugie

File: Pod/Classes/LSTracer.m [hunk 1]
```diff
@@ -338,30 +338,39 @@ - (void) _refreshImp {
     }
     if (sleepSecs > 0) {
         NSLog(@"LSTracer backing off for %@ seconds", @(sleepSecs));
-        [NSThread sleepForTimeInterval:sleepSecs];
```
Comment (bcronin) [inline Pod/Classes/LSTracer.m:341]: For future development, it'd be good to update the comment on line 323 to reflect these changes.

**Comment by bcronin** [comment]
Looks good

**Comment by eugie** [comment]
looks good! thanks!

**Pull request merged**